u-root.go

```diff
@@ -190,6 +190,13 @@ var (
 			"github.com/u-root/u-root/cmds/rmmod",
 			"github.com/u-root/u-root/cmds/sshd",
 			"github.com/u-root/u-root/cmds/switch_root",
+			"github.com/u-root/u-root/cmds/init",
+			"github.com/u-root/u-root/cmds/rush",
+			"github.com/u-root/u-root/cmds/cat",
+			"github.com/u-root/u-root/cmds/ls",
+			"github.com/u-root/u-root/cmds/pci",
+			"github.com/u-root/u-root/cmds/ip",
+			"github.com/u-root/u-root/cmds/dhclient",
```

Comment:
The list should be sorted, but looks good otherwise!

Reply:
Done